api-specificatie/openapi.yaml

```diff
@@ -167,37 +203,65 @@ paths:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/403"
         '406':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/406"
-        '409':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/409"
-        '410':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/410"
+        '500':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/500"
+        '503':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/503"
+        'default':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/default"
       tags:
-      - gewijzigdepersonen
+      - Raadplegen gewijzigde personen
 components:
+  parameters:
+    burgerservicenummer:
+      in: path
+      name: burgerservicenummer
+      required: true
+      description: "Identificerend gegeven van een ingeschreven natuurlijk persoon, als bedoeld in artikel 1.1 van de Wet\
+        \ algemene bepalingen burgerservicenummer."
+      schema:
+        pattern: "^[0-9]*$"
+        maxLength: 9
+        minLength: 9
+        example: "555555021"
   schemas:
-    VolgindicatiePersonenHalCollectie:
+    VolgindicatiesHalCollectie:
       type: object
       properties:
         _links:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/schemas/HalCollectionLinks"
         _embedded:
-          $ref: '#/components/schemas/VolgindicatiePersonen'
-    VolgindicatiePersonen:
+          $ref: '#/components/schemas/VolgindicatiesHal'
+    VolgindicatiesHal:
       type: object
       properties:
-        volgindicatiepersonen:
+        volgindicaties:
           type: array
           items:
-            $ref: '#/components/schemas/VolgindicatiePersoon'
-    VolgindicatiePersoon:
+            $ref: '#/components/schemas/VolgindicatieHal'
+    Volgindicatie:
       type: object
       properties:
-        burgerservicenummer:
```

Comment:
Waarom is deze verwijderd? Hierdoor kan alleen maar via de self link worden bepaald om welke persoon dit betreft

Reply:
Gedachte hierachter is dat er dan één consistente resource definitie is over alle operaties (PUT en GET). En in de PUT lijkt het mij niet handig daarin ook burgerservicenummer op te nemen, want die zit ook al in de url, met als risico dat ze verschillende zijn.

Alternatief zou denk ik zijn burgerservicenummer wel op te nemen in de resource en dan POST te gebruiken voor upsert in plaats van PUT. @MelvLee Is dat beter?

Reply:
Het is gebruikelijk om voor de POST/PUT/PATCH operaties een andere model te gebruiken dan voor de GET. Je wilt bij de GET vaak dat de data anders wordt gerepresenteerd dan hoe je de data presententeert bij het opvoeren van een wijziging. Denk bijv. aanschrijfwijze en gebruikInLopendeTekst properties bij IngeschrevenPersoon.
Ik zou dus voor 2 resource definities gaan, ook als we POST zouden kiezen voor de upsert. Dat is (volgens mij) meer evovable

Reply:
> ook als we POST zouden kiezen voor de upsert. Dat is (volgens mij) meer evolvable

Bedoel je hiermee dat de POST meer evolvable is dan de PUT? Of doel je hiermee op het onderscheid maken tussen PUT en GET resources?

Reply:
Nee, ik bedoel 2 resource definities, 1 voor creeren/updaten en 1 voor tonen

Reply:
ik heb het burgerservicenummer weer toegevoegd aan de responses